### ISSUE TITLE
Carbon Calculator: Accessibility - Ensure the order of headings is semantically correct

### ISSUE THREAD

**Issue by BrianGibbVS** [visitscotland/vs-component-library#244]
### Raised by
Brian Gibb

### Describe the bug
In the Carbon Calculator there are heading orders which are invalid. To meeting WCAG guidelines, heading levels should only increment by 1 at a time. Attached are examples where levels jump and fail accessibility scans.

### To Reproduce
1. Go to Carbon Calculator page
2. Conduct an Accessibility Scan on each of the Carbon Calculator steps.
3. See screenshots.

### Expected behavior
As per [WCAG guildlines](https://www.w3.org/WAI/tutorials/page-structure/headings/), heading levels should only increment by 1 at a time h2 h3 h4 etc and not and not skip levels like h2 h4.

### Page URL
https://www.visitscotland.com/travel-planning/responsible-tourism/carbon-calculator

### Environment
All

### Browser
All

### Device and version
N/A

### Impact on end user
Makes navigating a page difficult for screenreader users.

### Impact on product development
N/A

### Additional context
N/A
